pkg/ccl/backupccl/testdata/backup-restore/backup-dropped-descriptors

```diff
@@ -10,9 +10,12 @@ SET use_declarative_schema_changer = 'off';
 SET CLUSTER SETTING jobs.debug.pausepoints = 'schemachanger.before.exec';
 CREATE DATABASE d;
 CREATE TABLE d.foo (id INT);
+----
```

Comment:
did the diff for this data driven test accidentally creep into the first commit?

Reply:
yeah I think I got excited and added a `expect-pausepoint` subclause to `exec-sql`. This doesn't live in `exec-sql` in the final state though so I can revert this diff.

Reply:
fwiw in a future PR I'm going to make `exec-sql` default to `require.NoError()` and if a statement expects an error they will have to add an additional `exec-sql expects-error`. This is more canonical with the way all other datadriven drivers work today.